task/upload.go

```diff
@@ -538,7 +546,7 @@ func uploadTaskOutputLocations(tctx *TaskContext) ([]OutputName, []clients.Outpu
 	} else {
 		mp4 = OUTPUT_ONLY_SHORT
 	}
-	outputNames, outputLocations, err := outputLocations(outURL, OUTPUT_ENABLED, playbackId, mp4, playbackId, !isEncryptionEnabled(*tctx.Task.Params.Upload))
+	outputNames, outputLocations, err := outputLocations(outURL, OUTPUT_ENABLED, playbackId, mp4, playbackId, "", "", !isEncryptionEnabled(*tctx.Task.Params.Upload))
```

Comment:
Why do we pass empty strings here? Is it because we don't support fMP4 in Transcode API?

Reply:
The opposite but yes, this is for the normal upload task, we only support fmp4 for transcode API at the moment.

Reply:
👍